src/main/java/zd/zero/waifu/motivator/plugin/integrations/ErrorReporter.kt

```diff
@@ -136,4 +136,42 @@ class ErrorReporter : ErrorReportSubmitter() {
         if (edition != null) appName += " ($edition)"
         return Pair(appInfo, appName)
     }
+
+
+    fun getMinifiedConfig(): String {
```

Comment:
So this minifies the configuration which results to:
```
{"iwotde":"1","isme":"1","ismse":"1","iutme":"1","iutmse":"1","imme":"1","immse":"1","ise":"1","ididfm":"1","itme":"1","itse":"1","iime":"1","iise":"0","iecne":"1","iecse":"1","itim":"5","iaf":"1","ebf":"5","pof":"75","aec":"0,130","v":"203.4818.26","ui":"40b17bb7-4603-438e-b568-e92dc3a26d95","ism":"1"}

length: 304
```
Getting the first letter and the acronym, if there's a key clash, it would add an index (rare case).

It results in a bit cryptic message 😅but reduces the length massively. Initially, I want to implement it with `registerTypeAdapter` but this requires manually add the properties to be included in serialization based on type. I also avoided adding `@SerializedName` annotation to reduce the name field as this also affects the persisting state in the platform.

Now, I want to change this into something, if we can just avoid the JSON format at all, there would also be a limitation here, where the preferred character will contain values and can't reduce the length size anymore.